plugins/Marketplace/stylesheets/plugin-details.less

```diff
@@ -3,11 +3,6 @@
     text-align: left;
     line-height: 20px;
 
-    > .row > .col {
```

Comment:
This change should not be needed here anymore?

Removing the change should remove the visual changes of the old modals, and we can keep your other suggested change for DEV-18004.

Reply:
Yep, I will revert this, it was a leftover from the first iteration.

Reply:
Reverted.